src/AWS.Deploy.Orchestration/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("AWS.Deploy.CLI")]
```

Comment:
CLI should not need `InternalsVisibleTo`

Reply:
Fixed